Essentials/src/com/earth2me/essentials/Warps.java

```diff
@@ -69,13 +72,28 @@ public void setWarp(String name, Location loc) throws Exception {
         }
         conf.setProperty(null, loc);
         conf.setProperty("name", name);
+        if (user != null) conf.setProperty("lastowner", user.getBase().getUniqueId().toString());
         try {
             conf.saveWithError();
         } catch (IOException ex) {
             throw new IOException(tl("invalidWarpName"));
         }
     }
-
+    
+    @Override
+    public UUID getLastOwner(String warp) throws WarpNotFoundException, InvalidWorldException {
```

Comment:
Why do we need to throw an invalid world exception?

Reply:
hehe i dont ses this sorry ;)